homeassistant/components/qwikswitch/sensor.py

```diff
@@ -34,8 +34,10 @@ def __init__(self, sensor):
         sensor_type = sensor["type"]
 
         self._decode, self.unit = SENSORS[sensor_type]
-        if isinstance(self.unit, type):
-            self.unit = f"{sensor_type}:{self.channel}"
+        # this cannot happen because it only happens in bool and this should be redirected to binary_sensor
+        assert not isinstance(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)